docs/content/en/main/concepts/syncer.md

```diff
@@ -96,14 +96,40 @@ You can then create foo.bar in this workspace, or create an APIBinding in anothe
 To sync resource from another existing APIExport in the KCP server, run
 
 ```sh
-kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace|another-apiexport -o syncer.yaml
+kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
 ```
 
 Syncer will start syncing the resources in this APIExport as long as the physical clusters has the compatible API schemas.
```

Comment:
```suggestion
Syncer will start syncing the resources in this `APIExport` as long as the physical clusters has the compatible API schemas.
```

Reply:
Either "as long as the physical clusters have" or "as long as the physical cluster has"

Reply:
Why not even:

as long as the `SyncTarget` has compatible API schemas